app/models/membership.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class Membership < ApplicationRecord
+  belongs_to :organization, optional: true
```

Comment:
sounds like it should be required

Reply:
This is for case of user is admin of all organizations. It's `organization_id: 0`

Reply:
OK. I got confirm that no need to do admin of all organization with `organization_id: 0`.
I will change to required.